tests/src/Kernel/SmsFrameworkProviderTest.php

```diff
@@ -383,6 +388,7 @@ public function testEventsQueueIncoming() {
     $this->container->get('cron')->run();
 
     $expected[] = SmsEvents::MESSAGE_INCOMING_PRE_PROCESS;
+    $expected[] = 'incomingEvent'; // Plugin incoming event.
```

Comment:
https://www.drupal.org/node/2832601#comment-11809897: Comment 7 & 8 are pretty much why this is weird.